context/app/static/js/shared-styles/cards/SelectableCard/SelectableCard.tsx

```diff
@@ -32,7 +32,7 @@ function SelectableCard({
           <SelectableCardText variant="subtitle1" $colorVariant={colorVariant}>
             {title}
           </SelectableCardText>
-          <Checkbox checked={isSelected} onChange={(e) => selectItem(e)} value={cardKey} />
+          {selectItem && <Checkbox checked={isSelected} onChange={(e) => selectItem(e)} value={cardKey} />}
```

Comment:
This approach should work too:

```suggestion
          {selectItem && <Checkbox checked={isSelected} onChange={selectItem} value={cardKey} />}
```